src/Nest/ElasticClient.Indices.cs

```diff
@@ -137,13 +137,14 @@ internal IndicesNamespace(ElasticClient client): base(client)
 		/// <para></para>
 		/// <a href = "https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-create-index.html">https://www.elastic.co/guide/en/elasticsearch/reference/master/indices-create-index.html</a>
 		/// </summary>
-		public CreateIndexResponse Create(IndexName index, Func<CreateIndexDescriptor, ICreateIndexRequest> selector = null) => Create(selector.InvokeOrDefault(new CreateIndexDescriptor(index: index)));
```

Comment:
As you all can see from the code still typing, the co-domain of `Func` is contravariant; or in other words; it's not until the `Create` call resolves its argument that you have to pass the explicitly implemented interface in. Now, it is possible callers have previously done a Liskov on you to provide their own interface, or your mocks may have, but it's *very* unlikely users have since this is a builder fluent API.